app/test_main.py

```diff
@@ -1,3 +1,33 @@
 from app.main import get_human_age
 
-# write your code here
+
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repeating similar test cases. This will make your tests cleaner and more maintainable.